docs/event-dispatcher.md

```diff
@@ -76,8 +85,46 @@ Example:
       "microblock_sequence": "3",
       "microblock_hash": "0x9304fcbcc6daf5ac3f264522e0df50eddb5be85df6ee8a9fc2384c54274daaac",
       "microblock_parent_hash": "0x4893ab44636023efa08374033428e44eca490582bd39a6e61f3b6cf749b474bd"
+    },
+    {
+      "burnchain_op": {
+        "TransferStx": {
+          "block_height": 10,
```

Comment:
it's the burnchain block height

Reply:
It might make more sense to serialize this as `burn_block_height` instead, i.e., adding the rename field attribute to the `block_height` fields:

```rust
#[serde(rename = "burn_block_height")]
```